src/components/auth/user-auth.ts

```diff
@@ -0,0 +1,16 @@
+class UserAuth {
+  setUserAuthenticated = (isAuthenticated: boolean): void => {
+    sessionStorage.setItem('isAuthenticated', JSON.stringify(isAuthenticated));
+  };
```

Comment:
if you use class notation, better to write methods in function notation. Same for all methods
```suggestion
  setUserAuthenticated(isAuthenticated: boolean): void {
    sessionStorage.setItem('isAuthenticated', JSON.stringify(isAuthenticated));
  };
```